CHANGELOG.md

```
@@ -1,5 +1,53 @@
+### Features
+
+- Bump to holochain 0.7.0-dev.7 by @matthme in [#52](https://github.com/holochain/hc-spin/pull/52)
+- New argument --force-admin-ports which is passed to 'hc sandbox' (#50) by @mattyg in [#50](https://github.com/holochain/hc-spin/pull/50)
+  - Feat: new argument --force-admin-ports which is passed to 'hc sandbox'
+  - Docs: changelog
+  - Chore: lint
+
+### Documentation
+
+- Added documentation about how to update to a new holochain version by @matthme in [#51](https://github.com/holochain/hc-spin/pull/51)
+# Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)